### ISSUE TITLE
RuntimeError when running Samplescript without attention in the develop branch

### ISSUE THREAD

**Issue by amjltc295** [IBM/pytorch-seq2seq#181]
I got the same error as #109 for the develop branch when `use_attention` is set to False in `sample.py`:
```
RuntimeError: invalid argument 2: view size is not compatible with input tensor's size and stride (at least one dimension spans across two contiguous subspaces). Call
.contiguous() before .view(). at /pytorch/aten/src/THC/generic/THCTensor.cpp:226
```
However, the line ` predicted_softmax = function(...` in forwrad_step() in `model/decoder_rnn.py` disappears. I'm not sure how/where to solve this problem.

**Comment by amjltc295** [comment]
The related code is removed in commt adab17b8d7649de2be4989ca99e2674df6dc46bc

**Comment by amjltc295** [comment]
Fixed by #182
